### PR TITLE
Move system_info_node to prbt_support

### PR DESCRIPTION
**Pull request by SansoneG**
Created as per https://github.com/PilzDE/pilz_robots/pull/353#issuecomment-618282303

* Move system_info_node itself
* Move associated tests
* Add standard source-code related checks to prbt_support/CMakeLists.txt since this is the first source-code to be added
* Add new dependencies to prbt_support/package.xml
* Update Changelogs

Should we:
 [ ] Update any ReadMe?
 [ ] Remove the compiler flags that are commented out?

### CODE REVIEW

**Review by martiniil — CHANGES_REQUESTED** [review]
Looks good.

> Should we:
[ ] Update any ReadMe?

Please create a README.md for prbt_support and add a small description of the node under the headline `ROS API` analogue to https://github.com/PilzDE/pilz_robots/blob/melodic-devel/prbt_hardware_support/README.md

>[ ] Remove the compiler flags that are commented out?

Please uncomment all flags.

**Review by ct2034 — APPROVED** [review]

**Comment by cdoehn** [comment]
@martiniil Could you please review the README

File: prbt_support/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# ROS API
+
+## SystemInfoNode
+
+Logs important system information.
+
+### Used Services
+
+- /prbt/driver/get_object (canopen_chain_node/GetObject)
+  - Holds information about joint firmware.
```
Comment (martiniil) [inline prbt_support/README.md]:
```suggestion
  - Read CANOpen object holding firmware information.
```

File: prbt_support/CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+)
+
+#add_definitions(-Wall)
+#add_definitions(-Wextra)
+#add_definitions(-Wno-unused-parameter)
+#add_definitions(-Werror)
+#add_definitions(-std=c++11)
+#add_definitions(-Wconversion) # At least this line needs to be below find_package to keep
+                              #  the flag away from the gmock build
+#add_definitions(-Wpedantic)
```
Comment (martiniil) [inline prbt_support/CMakeLists.txt]:
```suggestion
add_definitions(-Wall)
add_definitions(-Wextra)
add_definitions(-Wno-unused-parameter)
add_definitions(-Werror)
add_definitions(-std=c++11)
add_definitions(-Wconversion) # At least this line needs to be below find_package to keep
                              #  the flag away from the gmock build
add_definitions(-Wpedantic)
```

**Review by martiniil — CHANGES_REQUESTED** [review]

**Review by martiniil — APPROVED** [review]

**Pull request merged**